bot_modules/irc_bot.js

```diff
@@ -105,6 +105,11 @@ command_parser = function(from, to, text, info) {
 		}
 	};
 
+	// make sure there's a string to parse! :X
+	if (typeof words[0] == 'undefined') {
```

Comment:
why not `if (words.length === 0) {` ? seems more idiomatic

Reply:
squashed into commit